src/api/spec/controllers/webui/project_controller_spec.rb

```diff
@@ -917,4 +917,65 @@
       it { expect(package.attribs.where(attrib_type: attribute_type)).to be_empty }
     end
   end
+
+  describe 'POST #new_release_request' do
+    let(:apache_maintenance_incident_project) { create(:maintenance_incident_project, name: 'ApacheMI') }
```

Comment:
With my comment I meant to drop the whole line because it is already defined in line 14.